lib/rpmsg/rpmsg.c

```diff
@@ -27,7 +27,7 @@ static uint32_t rpmsg_get_address(unsigned long *bitmap, int size)
 	unsigned int addr = RPMSG_ADDR_ANY;
 	unsigned int nextbit;
 
-	nextbit = metal_bitmap_next_clear_bit(bitmap, 0, size);
+	nextbit = metal_bitmap_next_clear_bit(bitmap, 1, size);
```

Comment:
endpoint address can be 0.

Reply:
The adress is allocated from 1 by the above change, so the LOCAL address is impossible to zero now.

Reply:
With you patch the address 0 is forbidden. So that means that the "well know" service will be forbidden if associated to address 0. I understand the simplification from a user point of view but the rpmsg_init_ept helper should answer to this.

Reply:
> With you patch the address 0 is forbidden. So that means that the "well know" service will be forbidden if associated to address 0. I understand the simplification from a user point of view but the rpmsg_init_ept helper should answer to this.

Ok, I will add a check in rpmsg_create_ept to return RPMSG_ERR_PARAM for zero local address.